config/config.go

```diff
@@ -2,3 +2,101 @@
 // Basic configuration like consul credentials and address, http port to listen for requests,
 // postgres schema name, credentials, and client timeout are read from environment variables.
 package config
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	consul "github.com/hashicorp/consul/api"
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Config model includes all necessary information, which will be read from environment variables
+type Config struct {
+	PostgresUser string `envconfig:"POSTGRES_USER" required:"true"`
+	PostgresPass string `envconfig:"POSTGRES_PASSWORD" required:"true"`
+	PostgresDB   string `envconfig:"POSTGRES_DB" required:"true"`
+
+	ConsulAddress string `envconfig:"CONSUL_ADDRESS" required:"true"`
+	ConsulToken   string `envconfig:"CONSUL_TOKEN" required:"true"`
+
+	BindIP   string `envconfig:"BIND_IP" default:"0.0.0.0"`
+	BindPort int    `envconfig:"BIND_PORT" default:"8000"`
```

Comment:
What are these values for?

Reply:
These values (IP address and Port) are using in ServerAddress() method and return address for http server... 
For example: _Server Listening at 0.0.0.0:8000..._

Reply:
So let them be HTTPIP and HTTPPort, then nobody will have any further questions to what are they.

Reply:
Good idea. I fixed it